ipatests/test_integration/test_automount_locations.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+"""
+This module provides tests for the automount location feature.
+"""
+
+from __future__ import absolute_import
+
+import time
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+# give some time for units to stabilize
+# otherwise we get transient errors
+WAIT_AFTER_INSTALL = 5
+WAIT_AFTER_UNINSTALL = WAIT_AFTER_INSTALL
+
+
+class TestAutomountInstallUninstall(IntegrationTest):
+    """
+    Test if vault feature behaves as expected, when KRA is installed or not
+    installed on replica
+    """
+
+    num_replicas = 1
+    topology = 'star'
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master, setup_dns=False)
+        client = cls.replicas[0]
+        tasks.install_client(cls.master, client)
+
+    def test_use_automount_location(self):
+
+        client = self.replicas[0]
+
+        self.master.run_command([
+            "ipa", "automountlocation-add", "baltimore"
+        ])
+
+        self.master.run_command([
+            "ipa", "host-mod", client.hostname,
+            "--location", "baltimore"
+        ])
+
+        result = client.run_command([
+            'ipa-client-automount', '--location', 'baltimore',
+            '-U', '--debug'
+        ])
+
+        assert result.returncode == 0
```

Comment:
You don't need to check the result code unless you use ``raiseonerr=True`` and expect an error.

Reply:
Thanks!
I've replaced that with a regular expression operation that checks for stderr=.+ in the debug output. It will detect non-fatal systemctl errors e.g. when systemctl fails but still returns 0. 
I do expect it to fail first and I'll update the test afterwards.